docs/source/tutorials/getting_started.rst

```diff
@@ -0,0 +1,43 @@
+.. title:: svgutils tutorial
```

Comment:
I think you will also need to add getting_started.rst to index.rst, so that it appears on the main documentaiton page..